src/lib.rs

```diff
@@ -3492,19 +3489,7 @@ pub type PFN_vkGetDeviceProcAddr =
                                                pName:
                                                    *const ::std::os::raw::c_char)
                               -> PFN_vkVoidFunction>;
-pub type PFN_vkCreateDevice =
```

Comment:
why removing the function declarations?

Reply:
They just bloat the file or are they needed for anything else?

Reply:
Will we need them for a loader-style initializer? Surely, we can always re-generate the header and bring those back, but then - let's either remove all of them or keep all of them.

Reply:
I see, so the goal is to have a ICD implementation. Then I'll revert those changes.